src/test/scala/org/kibanaLoadTest/helpers/Helper.scala

```diff
@@ -316,20 +322,21 @@ object Helper {
       new InputStreamReader(new FileInputStream(filePath.toFile))
     } else throw new RuntimeException(s"Cannot parse the file: $filePath")
     val br = new BufferedReader(inputStream)
-    var strLine = Option(br.readLine())
-    var jsonString = ""
     val jsonStringBuffer = new ArrayBuffer[String]()
+
+    var strLine = Option(br.readLine())
+    var sb = new StringBuffer("")
```

Comment:
Nit: `StringBuilder` offers the same functionality but better performance and is the preferred alternative.